src/dmd/backend/ty.d

```diff
@@ -166,6 +166,7 @@ enum
 
     // Used for symbols going in the __thread_data section for TLS variables for Mach-O 64bit
     mTYthreadData   = 0x5000,
+    mTYweakLinkage  = 0x6000,
```

Comment:
I don't think mTYweakLinkage is used for symbols going in the __thread_data section? Some description of it should be here.

Reply:
Fixed.